Pip Install Error in Docker Image

When running `pip install manifold3d` as part of building a docker image, I'm getting the following error:

`Dockerfile` contents as a MWE:
```
FROM gradescope/auto-builds:ubuntu-20.04

RUN pip3 install manifold3d
```

Output:
```
[+] Building 87.0s (6/6) FINISHED                                                                                                                                                                                                                                          docker:default
 => [internal] load build definition from Dockerfile                                                                                                                                                                                                                                 0.0s
 => => transferring dockerfile: 106B                                                                                                                                                                                                                                                 0.0s
 => [internal] load .dockerignore                                                                                                                                                                                                                                                    0.0s
 => => transferring context: 2B                                                                                                                                                                                                                                                      0.0s
 => [internal] load metadata for docker.io/gradescope/auto-builds:ubuntu-20.04                                                                                                                                                                                                       0.4s
 => [auth] gradescope/auto-builds:pull token for registry-1.docker.io                                                                                                                                                                                                                0.0s
 => CACHED [ 1/15] FROM docker.io/gradescope/auto-builds:ubuntu-20.04@sha256:c3a626b2b37d39404d0fa11dba3dbb2f2e4089b67e25544c750c5d9949a81349                                                                                                                                        0.0s
 => ERROR [2/2] RUN pip3 install manifold3d                                                                                                                                                                                                                                         86.6s
------                                                                                                                                                                                                                                                                                    
 > [2/2] RUN pip3 install manifold3d:                                                                                                                                                                                                                                                     
2.112 Collecting manifold3d                                                                                                                                                                                                                                                               
2.230   Downloading manifold3d-2.2.1.tar.gz (374 kB)                                                                                                                                                                                                                                      
2.380   Installing build dependencies: started                                                                                                                                                                                                                                            
3.807   Installing build dependencies: finished with status 'done'                                                                                                                                                                                                                        
3.808   Getting requirements to build wheel: started
3.861   Getting requirements to build wheel: finished with status 'done'
3.862   Installing backend dependencies: started
12.85   Installing backend dependencies: finished with status 'done'
12.85     Preparing wheel metadata: started
12.93     Preparing wheel metadata: finished with status 'done'
13.27 Collecting numpy; python_version < "3.12"
13.30   Downloading numpy-1.24.4-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (17.3 MB)
18.26 Building wheels for collected packages: manifold3d
18.27   Building wheel for manifold3d (PEP 517): started
86.15   Building wheel for manifold3d (PEP 517): still running...
86.41   Building wheel for manifold3d (PEP 517): finished with status 'error'
86.41   ERROR: Command errored out with exit status 1:
86.41    command: /usr/bin/python3 /tmp/tmpghrfs441 build_wheel /tmp/tmp53oy5eck
86.41        cwd: /tmp/pip-install-qglfur6p/manifold3d
86.41   Complete output (75 lines):
86.41   *** scikit-build-core 0.6.1 using CMake 3.27.7 (wheel)
86.41   *** Configuring CMake...
86.41   loading initial cache file /tmp/tmp30nqq3d6/build/CMakeInit.txt
86.41   -- The CXX compiler identification is GNU 9.4.0
86.41   -- Detecting CXX compiler ABI info
86.41   -- Detecting CXX compiler ABI info - done
86.41   -- Check for working CXX compiler: /usr/bin/c++ - skipped
86.41   -- Detecting CXX compile features
86.41   -- Detecting CXX compile features - done
86.41   -- Found Python: /usr/bin/python3 (found version "3.8.10") found components: Interpreter Development.Module
86.41   -- The C compiler identification is GNU 9.4.0
86.41   -- Detecting C compiler ABI info
86.41   -- Detecting C compiler ABI info - done
86.41   -- Check for working C compiler: /usr/bin/cc - skipped
86.41   -- Detecting C compile features
86.41   -- Detecting C compile features - done
86.41   Parallel Backend: TBB
86.41   CMake Deprecation Warning at /tmp/tmp30nqq3d6/build/_deps/glm-src/CMakeLists.txt:1 (cmake_minimum_required):
86.41     Compatibility with CMake < 3.5 will be removed from a future version of
86.41     CMake.
86.41   
86.41     Update the VERSION argument <min> value or use a ...<max> suffix to tell
86.41     CMake that the project does not need compatibility with older versions.
86.41   
86.41   
86.41   CMake Deprecation Warning at /tmp/tmp30nqq3d6/build/_deps/glm-src/CMakeLists.txt:2 (cmake_policy):
86.41     Compatibility with CMake < 3.5 will be removed from a future version of
86.41     CMake.
86.41   
86.41     Update the VERSION argument <min> value or use a ...<max> suffix to tell
86.41     CMake that the project does not need compatibility with older versions.
86.41   
86.41   
86.41   CMake Deprecation Warning at /tmp/tmp30nqq3d6/build/_deps/thrust-src/CMakeLists.txt:9 (cmake_policy):
86.41     The OLD behavior for policy CMP0104 will be removed from a future version
86.41     of CMake.
86.41   
86.41     The cmake-policies(7) manual explains that the OLD behaviors of all
86.41     policies are deprecated and that a policy should be set to OLD only under
86.41     specific short-term circumstances.  Projects should be ported to the NEW
86.41     behavior and not rely on setting a policy to OLD.
86.41   
86.41   
86.41   -- Could NOT find PkgConfig (missing: PKG_CONFIG_EXECUTABLE)
86.41   -- tbb not found, downloading from source
86.41   CMake Deprecation Warning at /tmp/tmp30nqq3d6/build/_deps/tbb-src/CMakeLists.txt:15 (cmake_minimum_required):
86.41     Compatibility with CMake < 3.5 will be removed from a future version of
86.41     CMake.
86.41   
86.41     Update the VERSION argument <min> value or use a ...<max> suffix to tell
86.41     CMake that the project does not need compatibility with older versions.
86.41   
86.41   
86.41   CMake Warning at /tmp/tmp30nqq3d6/build/_deps/tbb-src/CMakeLists.txt:118 (message):
86.41     You are building oneTBB as a static library.  This is highly discouraged
86.41     and such configuration is not supported.  Consider building a dynamic
86.41     library to avoid unforeseen issues.
86.41   
86.41   
86.41   -- Performing Test CMAKE_HAVE_LIBC_PTHREAD
86.41   -- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
86.41   -- Check if compiler accepts -pthread
86.41   -- Check if compiler accepts -pthread - yes
86.41   -- Found Threads: TRUE
86.41   -- TBBBind build targets are disabled due to unsupported environment
86.41   CMake Error at bindings/python/third_party/nanobind/CMakeLists.txt:34 (message):
86.41     The nanobind dependencies are missing! You probably did not clone the
86.41     project with --recursive.  It is possible to recover by invoking
86.41   
86.41     $ git submodule update --init --recursive
86.41   
86.41   
86.41   -- Configuring incomplete, errors occurred!
86.41   
86.41   *** CMake configuration failed
86.41   ----------------------------------------
86.41   ERROR: Failed building wheel for manifold3d
86.41 Failed to build manifold3d
86.46 ERROR: Could not build wheels for manifold3d which use PEP 517 and cannot be installed directly
------
Dockerfile:3
--------------------
   1 |     FROM gradescope/auto-builds:ubuntu-20.04
   2 |     
   3 | >>> RUN pip3 install manifold3d
--------------------
ERROR: failed to solve: process "/bin/sh -c pip3 install manifold3d" did not complete successfully: exit code: 1
```

--- a/.github/workflows/publish_npm.yml
+++ b/.github/workflows/publish_npm.yml
@@ -1,0 +1,25 @@
+name: publish_npm
+
+on: [release, workflow_dispatch]
+
+jobs:
+  deploy_github_pages:
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Download built examples
+      uses: dawidd6/action-download-artifact@v2
+      with:
+        workflow: manifold.yml
+        workflow_conclusion: completed
+        name: wasm
+        path: ./bindings/wasm/
+
+    - name: Publish to npm
+      run: |
+        cd ./bindings/wasm/
+        npm publish
+      env:
+        NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}

--- a/bindings/wasm/package.json
+++ b/bindings/wasm/package.json
@@ -1,6 +1,6 @@
 {
   "name": "manifold-3d",
-  "version": "2.2.1",
+  "version": "2.2.2",
   "description": "Geometry library for topological robustness",
   "main": "manifold.js",
   "files": [

--- a/flake.nix
+++ b/flake.nix
@@ -52,7 +52,7 @@
             }: pkgs.stdenv.mkDerivation {
               inherit doCheck;
               pname = "manifold-${parallel-backend}";
-              version = "2.2.1";
+              version = "2.2.2";
               src = self;
               nativeBuildInputs = (with pkgs; [
                 cmake
@@ -103,7 +103,7 @@
               parallelBackends)) // {
             manifold-js = pkgs.buildEmscriptenPackage {
               name = "manifold-js";
-              version = "2.2.1";
+              version = "2.2.2";
               src = self;
               nativeBuildInputs = (with pkgs; [ cmake python39 ]);
               buildInputs = [ pkgs.nodejs ];
@@ -134,7 +134,7 @@
             # but how should we make it work with other python versions?
             manifold3d = with pkgs.python3Packages; buildPythonPackage {
               pname = "manifold3d";
-              version = "2.2.1";
+              version = "2.2.2";
               src = self;
               propagatedBuildInputs = [
                 numpy

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "manifold3d"
-version = "2.2.1"
+version = "2.2.2"
 authors = [
   { name="Emmett Lalish", email="elalish@gmail.com" },
 ]

